### PR TITLE
Add get_config interface on serving.

### PR DESCRIPTION
**Pull request by gongweibao**
Add an rpc interface `get_config` on serving. So
1. client needs not to prepare the model config file.
2. client can get the `max_batch_size` from this server.

```
message ServingConfig{
    int32 max_batch_size = 1;
    string proto_txt = 2;
};

service MultiLangGeneralModelService {	
  rpc get_config(EmptyRequest) returns (ServingConfig) {}
};
```

### CODE REVIEW

File: core/configure/proto/multi_lang_general_model_service.proto [hunk 1]
```diff
@@ -43,8 +43,17 @@ message Response {
...
   optional string engine_name = 2;
-}
+};
+
+message EmptyRequest{
+};
+
+message ServingConfig{
+    int32 max_batch_size = 1;
+    string proto_txt = 2;
```
Comment (MRXLT) [inline core/configure/proto/multi_lang_general_model_service.proto]: Expected "required", "optional", or "repeated"

File: python/paddle_serving_client/__init__.py [hunk 2]
```diff
@@ -384,17 +385,29 @@ def release(self):
...
+        self._parse_model_config(config.proto_txt)
+
+    def connect(self, endpoint, use_remote_config=True):
         self.channel_ = grpc.insecure_channel(endpoint[0])  #TODO
-        self.stub_ = multi_lang_general_model_service_pb2_grpc.MultiLangGeneralModelServiceStub(
+        self.stub_ = grpc_pb2.MultiLangGeneralModelServiceStub(
             self.channel_)
 
+        if use_remote_config:
+            self._load_client_config(stub)
```
Comment (MRXLT) [inline python/paddle_serving_client/__init__.py]: 参数是否应该是self.stub_

File: python/paddle_serving_client/__init__.py [hunk 3]
```diff
@@ -384,17 +385,29 @@ def release(self):
...
-        self._parse_model_config(path)
+        with open(path, 'r') as f:
+            proto_txt = str(f.read())
+        
+        self._parse_model_config(proto_txt)
 
-    def connect(self, endpoint):
+    def _load_client_config(self, stub):
+        req= pb2.EmptyRequest()
+        self._config  = self.stub_.get_client_proto_text(req)
```
Comment (MRXLT) [inline python/paddle_serving_client/__init__.py]: 这里get_client_proto_text是否应该是get_config

File: python/paddle_serving_client/__init__.py (hunk 2 above)
Comment (gongweibao) [reply python/paddle_serving_client/__init__.py]: done!

File: python/paddle_serving_client/__init__.py (hunk 3 above)
Comment (gongweibao) [reply python/paddle_serving_client/__init__.py]: done!

File: core/configure/proto/multi_lang_general_model_service.proto (hunk 1 above)
Comment (gongweibao) [reply core/configure/proto/multi_lang_general_model_service.proto]: done!

**Pull request closed without merging**